### PR TITLE
add aws/gce volume id to kube_persistentvolume_info

### PR DESCRIPTION
**Pull request by povilasv**
ref https://github.com/kubernetes/kube-state-metrics/issues/704

### CODE REVIEW

File: internal/store/persistentvolume.go [hunk 1]
```diff
@@ -102,11 +102,32 @@ var (
...
+				switch {
+				case p.Spec.PersistentVolumeSource.GCEPersistentDisk != nil:
+					volLabelIDKey = "gce_persistent_disk_name"
+					volLabelIDValue = p.Spec.PersistentVolumeSource.GCEPersistentDisk.PDName
+				case p.Spec.PersistentVolumeSource.AWSElasticBlockStore != nil:
+					volLabelIDKey = "ebs_volume_id"
+					volLabelIDValue = p.Spec.PersistentVolumeSource.AWSElasticBlockStore.VolumeID
+				}
+
+				if volLabelIDKey == "" {
```
Comment (brancz) [inline internal/store/persistentvolume.go:115]: Prometheus will drop the empty value anyways, so why the extra branch?

Comment (povilasv) [reply internal/store/persistentvolume.go]: :+1: I'm not sure about empty keys, so I default key to `default`

**Comment by brancz** [comment]
Even the key doesn’t need to be conditional does it? We just add all of them always.

**Comment by povilasv** [comment]
Agreed, thanks for the tip. I did the change :+1:

**Comment by brancz** [comment]
Very nice.

/lgtm
/approve

**Comment by povilasv** [comment]
CI was failing due to missing docs, so I have fixed it.

```
=== RUN   TestDocumentation
    TestDocumentation: main_test.go:133: Label gce_persistent_disk_name not found in documentation. Documented labels for metric kube_persistentvolume_info are: persistentvolume, storageclass
    TestDocumentation: main_test.go:133: Label ebs_volume_id not found in documentation. Documented labels for metric kube_persistentvolume_info are: persistentvolume, storageclass
--- FAIL: TestDocumentation (0.03s)
```

**Comment by brancz** [comment]
Isn't it lovely when tests work :smile: 

/lgtm

**Comment by povilasv** [comment]
it is :D That doc test is pretty awesome.

**Pull request merged**